files/usr/share/cinnamon/cinnamon-settings/modules/cs_themes.py

```diff
@@ -69,6 +69,18 @@ def on_module_selected(self):
             page = SettingsPage()
             self.sidePage.stack.add_titled(page, "options", _("Settings"))
 
+            settings = page.add_section(_("Desktop color options"))
+
+            invert = lambda b: not b
+            widget = GSettingsSwitch(_("Use default colors"), "org.cinnamon.theme.colors", "enable", map_get=invert, map_set=invert)
```

Comment:
Also note this. I wasn't sure if this was the best way to do it. I could have used `not_` from the operator module, but I didn't want to add an import.